src/sentry/utils/data_scrubber.py

```diff
@@ -78,7 +78,7 @@ def apply(self, data):
             self.filter_stacktrace(data['stacktrace'])
 
         if 'exception' in data:
-            for exc in data['exception']['values']:
+            for exc in data['exception'].get('values') or ():
```

Comment:
Please use `get_path(data, 'exception', 'values', filter=True, default=())` here for consistency